src/main/java/uk/ac/ebi/ampt2d/accession/file/UuidFile.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.file;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Table(name = "uuid_file")
```

Comment:
Isn't this the default? Or would it be in plural?

Reply:
It is possible that it is the default, I think I told Selva to set the table name with a specific value to avoid changing it every time the class name changes, although it should probably be in plural.